modules/zip/doc_classes/ZIPReader.xml

```diff
@@ -14,6 +15,32 @@
 		    var res := reader.read_file("hello.txt")
 		    reader.close()
 		    return res
+
+		# Extract all files from a ZIP archive, preserving the directories within.
+		# This acts like the "Extract all" functionality from most archive managers.
+		func extract_all_from_zip():
+		    var reader := ZIPReader.new()
```

Comment:
Side note, does this mean that examples using inferred types with `:=` are accepted in class reference examples moving forward? I'm all for static typing in examples, and for inferred types as a compromise in the class reference specifically.

Reply:
We have no agreement on `:=` specifically but static typing is still not ideal in the class reference. Calinou is steadily bringing it to the manual and surrounding tutorials, though.

Reply:
We only have 4 classes which use `:=`:
```
$ rg -g'*.xml' " := "
modules/zip/doc_classes/ZIPReader.xml
10:                 var reader := ZIPReader.new()
11:                 var err := reader.open("user://archive.zip")
14:                 var res := reader.read_file("hello.txt")

modules/zip/doc_classes/ZIPPacker.xml
10:                 var writer := ZIPPacker.new()
11:                 var err := writer.open("user://archive.zip")

doc/classes/DTLSServer.xml
14:             var dtls := DTLSServer.new()
15:             var server := UDPServer.new()
93:             var dtls := PacketPeerDTLS.new()
94:             var udp := PacketPeerUDP.new()

doc/classes/UDPServer.xml
16:             var server := UDPServer.new()
80:             var udp := PacketPeerUDP.new()
```

IMO we should stick to the consensus for now and not use typing in the docs.

If we want to start using typing, we need to have a proper plan for what should be typed, how to update existing content over time, and write all this in guidelines. So for now, I'd go as far as changing those 4 classes to follow the current convention.

Reply:
I went ahead and replaced `:=` with `=`.